patches/chrome-browser-shell_integration.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
+index 29e98f5b2839a44a488d6baec756b660935302dd..b761dea9fbb8f0009976997a41d9a10b2aaf3df1 100644
+--- a/chrome/browser/shell_integration.h
++++ b/chrome/browser/shell_integration.h
+@@ -235,6 +235,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
+  protected:
+   ~DefaultBrowserWorker() override;
+ 
++  BRAVE_DEFAULT_BROWSER_WORKER_H
```

Comment:
these should always go in public

Reply:
ha! I actually asked Simon to move it before `private:` (which he did), but missed the part that this class have `protected:` (kinda rare in Chromium). Should be more precise with wording 🙂 https://github.com/brave/brave-core/pull/11125#discussion_r751259211